Multisig/UI/App/UIKit/UI Library/ReusableCells/AddressDetailsCell.swift

```diff
@@ -9,49 +9,30 @@
 import UIKit
 
 class AddressDetailsCell: UITableViewCell {
-    @IBOutlet private weak var addressWithTitleView: AddressWithTitleView!
+    @IBOutlet private weak var addressInfoView: AddressInfoView!
 
-    private var style: Style = .nameAndAddress
     var onViewDetails: (() -> Void)?
 
     static let rowHeight: CGFloat = 68
-    private var address: Address?
-
-    enum Style {
-        case nameAndAddress
-        case address
-    }
 
     override func awakeFromNib() {
         super.awakeFromNib()
-        addressWithTitleView.addTarget(self, action: #selector(copyAddress), for: .touchUpInside)
-    }
-
-    @IBAction func viewAddressDetails() {
-        onViewDetails?()
+        addressInfoView.setDisclosureButtonImage(#imageLiteral(resourceName: "ico-browse-address"))
+        addressInfoView.onAddressInfoSelection = copyAddress
+        addressInfoView.onDisclosureButtonAction = viewDetails
     }
 
-    func setAddress(_ value: Address) {
-        addressWithTitleView.setAddress(value)
+    func setAddressInfo(_ addressInfo: AddressInfo) {
+        addressInfoView.setAddressInfo(addressInfo)
     }
 
-    func setName(_ value: String) {
```

Comment:
I thik the code will be easier to read on the call site if we provide additional helper methods

```
setAddress(_ address: Address, label: String? = nil)
setAddress(_ address: String, label: String? = nil)
```

Reply:
this cell is refactored into DetailAccountCell with one method `func setAccount(addressInfo: AddressInfo, title: String?)`
I think there is no need in helper methods at this point of time.